chainerio/container.py

```diff
@@ -7,10 +7,12 @@
 class Container(IO):
 
     def __init__(self, base_handler: IO, base: str,
-                 io_profiler: Optional[IOProfiler] = None, root: str = ""):
+                 io_profiler: Optional[IOProfiler] = None, root: str = "",
+                 eager_open=False):
```

Comment:
Could you explain the possible use case where `eager_open=False` is needed or must be done within the library?  For example, [ZippedImageDataset](https://github.com/chainer/chainer/blob/v6/chainer/datasets/image_dataset.py#L247) already has workaround on issues at MultiprocessIterator and that isn't much complicated. I still don't want to add another keyword argument to container init interface.

Reply:
One example I think we just encountered was: in case of nested zip with Python < 3.7, if we always open zip in `__init__`, i.e. always `eager_open=True` , according to this [PR](https://github.com/chainer/chainerio/pull/38#discussion_r311346659) and our solution in #40 , the whole zip file would be read when creating the handler, as we discussed in that PR, which can be far away from where the first file in zip is actually used. 

I understand your concern about having additional argument in init. This solution can be unnecessary.

Reply:
Ah I see. Let's finish #40 first and then come back here.

Reply:
A nit, but method arguments like this should be in `**kwargs` .